js/adapt-contrib-resourcesView.js

```diff
@@ -50,7 +50,7 @@ define([
         onResourceClicked: function(event) {
             var data = $(event.currentTarget).data();
 
-            if (!data.forceDownload) {
+            if (!data.forceDownload || Adapt.device.OS.toLowerCase() == 'ios') {
```

Comment:
Could you remove the `.toLowerCase()` from here and from line 60 please?

Reply:
And switch to strict equality? `===`

Reply:
Done!